cli/flox-activations/src/activations.rs

```diff
@@ -30,7 +98,7 @@ struct Activation {
     /// and use as a directory name.
     id: Uuid,
     /// The store path of the built environment
-    store_path: String,
+    store_path: PathBuf,
```

Comment:
nonblocking: we have `store_path: String` in 3 other spots throughout the codebase so I think we should stick with that for consistency